test/contracts/pass/cheatCodes.sol

```diff
@@ -123,7 +127,56 @@ contract CheatCodes is DSTest {
         assertEq(prankster.prankme(), address(this));
     }
 
-    function test_prank_val() public {
+    // this is not supported yet due to restrictions around symbolic address aliasing...
+    function prove_deal_unknown_address(address e, uint val) public {
+        hevm.deal(e, val);
+        assert(e.balance == val);
+    }
```

Comment:
```suggestion
    // TODO this is not supported yet due to restrictions around symbolic address aliasing...
    /*function prove_deal_unknown_address(address e, uint val) public {
        hevm.deal(e, val);
        assert(e.balance == val);
    }*/
```

Reply:
This is failing in CI, so perhaps we can just take it out, since it's a known issue, and add it as a TODO?

Reply:
prefixed it with `proveFail` for now

Reply:
Hmmm that can be a bit misleading, but it's OK!